assignment/scripts/loops-practice.js

```diff
@@ -32,16 +44,42 @@ let stars = ['Polaris', 'Gacrux', 'Formalhaut', 'Rigel', 'Deneb']
 
 // 2.a. TODO: Write a for of loop to console.log each star in the 'stars' array
 console.log('Some stars:');
+for (let star of stars) {
+  console.log(star);
+}
 
 
 // 3. While loops  
 console.log('---- 3. While loop ----');
 
 // 3.a. TODO: Write a while loop to console.log each star in the 'stars' array
 console.log('Some stars using while:');
+// this method is simple but wipes out the array
```

Comment:
Definitely a good thing to be aware of. In this assignment you can get away with it because you know you will never need `stars` again. However, as a general rule of thumb, mutating arrays should be done rarely and in a very limited scope (for example with an array that was created inside a small function).